api/helpers/uploadedFileErrorChecks.js

```diff
@@ -30,13 +30,17 @@ async function connectToClamAV(req) {
 }
 
 function initialiseClamScan(req) {
-    const { clamav_host: clamavHost, clamav_port: clamavPort } =
-        req._sails.config.upload;
-
+    const {
+        clamav_host: clamavHost,
+        clamav_port: clamavPort,
+        clamav_debug_enabled: clamavDebugEnabled,
```

Comment:
Can you make it so clamav_debug_enabled is defaulted to false?

Reply:
Sure I can do that. Ideally, the `false` value should be coming from the .env file but having a hardcoded `false` could act as a fallback.